apps/finance/contracts/Finance.sol

```diff
@@ -51,6 +51,7 @@ contract Finance is AragonApp {
         uint256 amount;
         uint256 paymentId;
         string reference;
+        uint256 paymentRepeatNumber;
```

Comment:
A very picky one, I would put this one line above, under `paymentId`, for consistence with the rest of the code. ;-)

Reply:
Ah—this is the tricky bit; we can't do that safely if we want to ensure people can upgrade up, since the underlying storage slots would stay in the previous order, but we'd have inserted a new element.

Doing it this way wouldn't break access to other variables, but would mean that existing transactions would all have a repeat number of 0.